### ISSUE TITLE
Mozilla Turkey logo

### ISSUE THREAD

**Issue by selimsumlu** [mozilla/OpenDesign#66]
Hi,

We're in the process of renaming our social media accounts and we might benefit from a new logo for Turkish community. Any help is appreciated. Thanks!
## Goal:

A new logo for "Mozilla Türkiye", the Turkish Mozilla community.
## Info:

It will be used on www.mozilla.org.tr and and our social media accounts (currently Facebook and Twitter).
## Style Information:

Text on logo will be **Mozilla Türkiye**.
Using the Turkish flag is acceptable (but not required). Using the star and crescent from the flag is also acceptable.
Colors are red (#E30A17) and white. See the Wikipedia entry for color details: https://en.wikipedia.org/wiki/Flag_of_Turkey#Colours
## Deadline:

No deadline, but ideally before the end of may.
## Tag:

[Design Needed]

**Comment by ioana-chiorean** [comment]
Hey @selimsumlu - glad to hear from you! I hope we will be able to create a cool logo! :tada:

**Comment by elioqoshi** [comment]
I can take this on! Might need some time but please ping me here again if I get caught up :)

**Comment by thomasjosephgreco** [comment]
@selimsumlu is this still something that you need ? I'm not sure if @elioqoshi was able to get around to it, but if not, i'd be happy to help out 😄

**Comment by selimsumlu** [comment]
Hi @thomasjosephgreco,
Yes, it wasn't an urgent thing we still need it.

**Comment by thomasjosephgreco** [comment]
@selimsumlu I can definitely help you out then 😄 Are there any color, or font schemes that you'd like me to follow, or do I kind of have a free-reign ? Nonetheless I am excited to help out with this !

**Comment by selimsumlu** [comment]
Thanks, Tom!

Please see the original post for suggested colors. Other than that, no rules/requirements to follow.

**Comment by elioqoshi** [comment]
Sorry about this, ended up in the backburner :(
I will help Tom with this :)

**Comment by parag9d** [comment]
Hi, @thomasjosephgreco any update on the logo?

**Comment by thomasjosephgreco** [comment]
hey @parag9d I have created a few different designs for this. I worked with @elioqoshi a little bit in the beginning - he gave me feedback on the building and I went from there. You might be aware but this is my first logo for mozilla so i'm interested in hearing any feedback you might have and I can re-work anything if necessary :) 

 
![mozilla-turkey-building](https://cloud.githubusercontent.com/assets/6646552/18372145/3e6ab6e4-7608-11e6-8de6-5647c3796a89.png)
![mozilla-turkey-shape2](https://cloud.githubusercontent.com/assets/6646552/18372144/3e665fe0-7608-11e6-96cc-f9c0ce9a659a.png)
![mozilla-turkey-shape4](https://cloud.githubusercontent.com/assets/6646552/18372143/3e4c7774-7608-11e6-9b60-8d78f4e5637c.png)

**Comment by parag9d** [comment]
@thomasjosephgreco There are issues with alignments of the Mozilla and Türkiye texts, probably making them left-aligned will give a better visual balance. Also, the font size and weight of Türkiye is too less to comprehend when used at a smaller size.

The spacing between the illustration and the text in the logo is a less. And, the height and horizontal alignment of the illustrations (on the left) is not visually centered with the Mozilla Türkiye text.

Kindly see if you can improve on these and if possible, please share the thoughts behind using those illustrations for the logo. I am unaware of how they symbolise Türkiye :)

**Comment by thomasjosephgreco** [comment]
@parag9d ok so i made some changes. I am curious though, what wordmark do you use when creating graphics? I have used the .eps and .png images but they don't make it easy for me to align text because the box surround the wordmark is quite large. 

Anyways, here are some .png files for the re-vamped logos. 
* I aligned the text to the left 
* increased the font-weight for turkey
* made the space between the wordmark and "turkey text" smaller.
* then horizonally centered the image 

Again, illustrator was not making it easy to align with the wordmark so let me know if there are issues, please let me know if there's a file I can use that makes aligning things easier. thanks :) 

![mozilla-turkey-building](https://cloud.githubusercontent.com/assets/6646552/18401359/2fff917e-76a8-11e6-9353-c7bef3f7b48e.png)
![mozilla-turkey-shape color](https://cloud.githubusercontent.com/assets/6646552/18401360/30010f54-76a8-11e6-9935-75e8b8afc17b.png)
![mozilla-turkey-shape1](https://cloud.githubusercontent.com/assets/6646552/18401361/3001c886-76a8-11e6-94d8-0595fe08629b.png)

**Comment by selimsumlu** [comment]
@thomasjosephgreco, thanks for volunteering on this.

The image below is what we currently use. I've only created it for a temporary use and it's far from perfect but might give you an idea about font sizes.

What I see as a more important issue is that the illustrations you provided don't seem to be representing Turkey. Did you have anything in mind when choosing/creating them? I'd like to know that as well. :)

![13217553_1051408714933311_4082091384263619580_o](https://cloud.githubusercontent.com/assets/1046789/18402080/68e13a9a-76e6-11e6-96b6-cf8910aed1bc.png)

**Comment by thomasjosephgreco** [comment]
@selimsumlu hey Selim, I have a handful of illustrations I put together. I really want to capture turkey as best as possible in this so i'd definitely be down to chat with you about it. Initially, I was going to create a detailed mosque, but wanted to stay away from religion to my illustration ended up being just that building. As for the shapes, I was basing it off of other mozilla logos - some of which just had pretty neat looking graphics. Please do let me know what you have in mind though as I would be happy to brain storm some ideas 😄

**Comment by Tess-welhazi** [comment]
I have a suggestion. I wanted something iconic about turkish culture and the hat is the first thing i could think of. I have a turkish friend who suggested drawing a grey wolf but i wasn't happy with the result.
This is my first contribution here and i could use any feedback i can get.

![image](https://cloud.githubusercontent.com/assets/20952652/20426443/6878aef6-ad7f-11e6-8976-e4f627ff88a9.png)

For the record this is how my wolf logo ended up looking
![image](https://cloud.githubusercontent.com/assets/20952652/20426893/7292bdb2-ad81-11e6-8751-3cd649598edf.png)

**Comment by selimsumlu** [comment]
I'm sorry people, I somehow lost track of this issue.

I strongly believe that we must stay clear of religious, ideological and political symbols. And this rules at the mosque, the fez (hat) and the gray wolf. I appraciate your hard work but all three of these symbols favor certain communities while leaving out others, and we don't want that.

In fact, this was the reason I didn't mention any symbols except the Turkish flag in my original post, because I couldn't think of any. Your time is precious, so if you have a creative idea, let's discuss it here even before drafting it.

**Comment by Tess-welhazi** [comment]
Alright, how about using the firefox or red panda mascots with the turkish flag? I'll share some sketches as soon as possible.

**Comment by selimsumlu** [comment]
@Tess-welhazi Those should be fine.

**Comment by Tess-welhazi** [comment]
How about this one?

![image](https://cloud.githubusercontent.com/assets/20952652/21390302/97e308be-c786-11e6-81b6-6b416ac2e99d.png)
this one is just a draft at this point i didn't recreate it in inkscape so the crescent and star are hand drawn

**Comment by Tess-welhazi** [comment]
so like this? 

![image](https://cloud.githubusercontent.com/assets/20952652/21698958/cc6cc8e4-d399-11e6-8d30-281066b9fcfc.png)

**Comment by selimsumlu** [comment]
Yes, it looks better to me.

**Comment by parag9d** [comment]
I love the version with Red Turkiye. Try to increase the space between the fox illustration and the texts. Also, reduce the size of the fox, like in bottom one.

**Comment by Tess-welhazi** [comment]
so like this
![image](https://cloud.githubusercontent.com/assets/20952652/21903698/27f86fee-d901-11e6-95d6-5ad90624c934.png)

**Comment by elioqoshi** [comment]
The spacing in the version before was fine, no need to increase the gap with the wordmark. 
Also, please use Fira Sans For Turkiye for consistency purposes.

Other than that, love it!

**Comment by Tess-welhazi** [comment]
changed the font and spacing again 
![image](https://cloud.githubusercontent.com/assets/20952652/22225230/f35c49c6-e1c1-11e6-8b46-1848fc7f2f56.png)

**Comment by barrosgeraldo** [comment]
It is necessary to update the proposal here with the new [Mozilla branding guidelines](https://mozilla.ninja/). I've made a first proposal based on [primary logo lockup](https://mozilla.ninja/expression/).

<img width="1437" alt="mozilla turkey logo 66 2x" src="https://user-images.githubusercontent.com/14970832/34924197-1a860390-f989-11e7-96b4-0eb48d18be41.png">

**Comment by pransh15** [comment]
So, I tinkered with few concepts I had in mind, and have come up with these. Let me know if something's wrong or something you'd want for me to change. I just tried to put pieces together.

* Based on the Turkey Flag:
![mozturkey1](https://user-images.githubusercontent.com/18158596/35044370-25204dcc-fbb6-11e7-84a2-a1a4358b030f.png)

* Based of Istanbul Skyline:
![mozturkey2](https://user-images.githubusercontent.com/18158596/35044428-580485a0-fbb6-11e7-922a-8868bce1d46c.png)
![mozturkey2a](https://user-images.githubusercontent.com/18158596/35044430-5843b388-fbb6-11e7-9c0f-a8c7ea02f006.png)
![mozturkey2b](https://user-images.githubusercontent.com/18158596/35044431-5881f026-fbb6-11e7-823e-8632473a2aed.png)

* Istanbul Skyline Concept, Multi-color:
![mozturkey3](https://user-images.githubusercontent.com/18158596/35044457-72bc720e-fbb6-11e7-90f4-0ddb4504c1e9.png)

**Comment by barrosgeraldo** [comment]
@pransh15, awesome, please, check the branding application guidelines to align this one under branding rules: https://mozilla.ninja/expression/

**Comment by pransh15** [comment]
@barrosgeraldo sure, I'll get back to you soon, with the updated graphics.

**Comment by selimsumlu** [comment]
Thanks for the updates, @barrosgeraldo and @pransh15. Do we know if the guidelines are finalized yet? (@elioqoshi)

I was thinking of something on the lines of these samples, which would have been much easier if we are provided with template files. But I'm not even sure if they align with the current guidelines.
https://static1.squarespace.com/static/58879499725e2572b9c4e0ec/t/592dff24ebbd1ad95610d015/1496186684791/?format=750w
https://static1.squarespace.com/static/58879499725e2572b9c4e0ec/t/592dfe1eb3db2bf8af733d4a/1496186413997/?format=750w

**Comment by pransh15** [comment]
![mozturkey4](https://user-images.githubusercontent.com/18158596/35058834-296096fc-fbdf-11e7-80c4-8426d574b0a0.png)
Somewhat like you asked.

**Comment by barrosgeraldo** [comment]
@selimsumlu I think will not have special branding guidelines for communities, it has taken a long time since the new brand was released and nothing has been reported about it, so I encourage you to keep using the [current guidelines](https://mozilla.ninja/expression/). [For communities, the appropriate framework is this](https://static1.squarespace.com/static/58879499725e2572b9c4e0ec/t/59cadd8a914e6b1221027674/1506467221270/Lockup-1Box.png?format=500w).

**Comment by pransh15** [comment]
![mozturkey4a](https://user-images.githubusercontent.com/18158596/35097449-288ead8a-fc76-11e7-8f34-b9522fc38b01.png)

So, this is one which is in line with the guidelines.

**Comment by kindavishal** [comment]
Hey @selimsumlu thanks for the issue!
I checked https://www.mozilla.org.tr/ it seems there is already a logo for the community, but that doesn't meet the [Branding Guidelines](https://mozilla.design) and it's necessary to update the logo as per the new guidelines.
Comments from https://github.com/mozilla/OpenDesign/issues/66#issuecomment-357566250 contain the logos, which meet the branding guidelines, please choose from the above.

**Comment by selimsumlu** [comment]
> 
> 
> Hey @selimsumlu thanks for the issue!
> I checked https://www.mozilla.org.tr/ it seems there is already a logo for the community, but that doesn't meet the [Branding Guidelines](https://mozilla.design) and it's necessary to update the logo as per the new guidelines.

There's actually no logo on www.mozilla.org.tr. I believe you meant the title, but it's just standard typography from the official [Frontierline theme](https://github.com/craigcook/frontierline-theme).

We would like to thank everyone involved for their suggestions and mockups. We appreciate your contributions, however we realised that it's quite hard to come up with a pleasing design while taking Turkey's inclusion matters into consideration.

We have decided to stick to the no-logo approach for now, so I'm going to close this issue.
